printer_mods/JosAr/Klicky-Probe/STL/README.md

```diff
@@ -0,0 +1,23 @@
+STL files
+
+There are no need for supports, recommended settings are 4 perimeters/top/bottom, 13% infill.
+All the files are ready to print, they are on the recommended orientation.
+
+- Version 0.1 - initial release
+- Version 0.2 - Revised probe and dock to improve the docking fit, remaining components are compatible
+  - Thanks to oc_geek for improving the probing of a badly tilted gantry (10º) and the probe and dock attachment, it's much better now
+  
+The Klicky Probe consists on four different components:
+- Afterburner (AB) mount
+- the probe itself
+- probe dock
+
+![Klicky Probe detail](Photos/Klicky_Probe detail.png)
```

Comment:
Please remove the space from the image file, for example by renaming it to `Klicky_Probe_detail.png`.

The relative image reference is incorrect, since this file is in the STL directory.  It should be something like

    ![Klicky Probe detail](../Photos/Klicky_Probe_detail.png)